buildSrc/src/main/java/Dependencies.kt

```diff
@@ -26,7 +26,10 @@ object Config {
      */
     val buildValues = listOf(
         "String", "API_URL",      "some.api.com/",
-        "String", "API_PROTOCOL", "https"
+        "String", "API_PROTOCOL", "https",
+
+        "long", "HTTP_READ_TIMEOUT", "60",
```

Comment:
Can we move `60` to some constant?

Reply:
Where would you like to put it?

Reply:
I'm not sure. What about object property/val - something like versions in Version object. Maybe extra object with constants?

Reply:
I'm not fond of this to be honest. That's 3 levels of indirection. I would leave build config as it is.